client/src/components/Setup/index.tsx

```diff
@@ -114,7 +114,7 @@ export default class Setup extends React.Component<any, ISetupState> {
       protocol = 'http';
     }
 
-    return protocol + '//' + host + '/auth/openid/return';
+    return protocol + '://' + host + '/auth/openid/return';
```

Comment:
The protocol var is set to 'https' without a colon here: https://github.com/lawrencegripper/ibex-dashboard/blob/3d71f1d27b35b526796b113b0cda9dae01b9c927/client/src/components/Setup/index.tsx#L109

Without the added colon the resulting url is "https//examplehost.com/auth/openid/return"